tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/test-union-type-with-null-literal/type-export/test_type.bal

```diff
@@ -0,0 +1,5 @@
+public type TestType string|null;
```

Comment:
Shall we add one more tests, like importing `TestType` from a different module?

Reply:
@rpjayasekara sorry, ignore this comment. It is already in a different module right?

Reply:
Yeah